modules/core/03-connection/types/msgs_test.go

```diff
@@ -132,12 +132,12 @@ func (suite *MsgTestSuite) TestNewMsgConnectionOpenTry() {
 		msg     *types.MsgConnectionOpenTry
 		expPass bool
 	}{
-		{"non empty connection ID", &types.MsgConnectionOpenTry{"connection-0", "clienttotesta", any, counterparty, 500, []*types.Version{ibctesting.ConnectionVersion}, clientHeight, suite.proof, suite.proof, suite.proof, clientHeight, signer}, false},
+		{"non empty connection ID", &types.MsgConnectionOpenTry{"connection-0", "clienttotesta", any, counterparty, 500, []*types.Version{ibctesting.ConnectionVersion}, clientHeight, suite.proof, suite.proof, suite.proof, clientHeight, signer, nil}, false},
```

Comment:
Just noticed this and can be done in a follow-up PR, but is there a reason why we cannot use `NewMsgConnectionOpenTry` as with the other test cases?

Reply:
Yeah, the constructor func doesn't set `PreviousConnectionId` which is being set here to force a failure. That field is deprecated so it must be empty in `ValidateBasic`